### PR TITLE
[FLINK-16098] [chinese-translation, Documentation] Translate "Overvie…

### PR DESCRIPTION
**Pull request by liuzhixing1006**
Translate "Overview" page of "Hive Integration" into Chinese
jira：https://issues.apache.org/jira/projects/FLINK/issues/FLINK-16098?filter=myopenissues
previously pull request：https://github.com/apache/flink/pull/11391

Sorry there was a problem with the previous branch, so I resubmitted a pullrequest
Thanks~

@JingsongLi @lirui-apache @wuc

### CODE REVIEW

**Comment by liuzhixing1006** [comment]
Hi @JingsongLi @lirui-apache I have finished the modification, Is there anything else I need to do? Thanks~

**Comment by lirui-apache** [comment]
Hi @liuzhixing1006 , I think some of my comments in #11391 are not addressed. Could you please double check? Note that comments might be hidden by GitHub, in which case you need to click `Load more...` to see them.

**Comment by liuzhixing1006** [comment]
Hi @lirui-apache, I'm sorry it was an oversight on my part, I have changed the missing parts and committed again.

**Comment by JingsongLi** [comment]
@flinkbot run travis
@flinkbot run azure

**Comment by liuzhixing1006** [comment]
Hi @lirui-apache , this pr has not been merged for so long. Is there anything that needs to be modified? Thanks.

**Comment by lirui-apache** [comment]
@JingsongLi Can you help with the merge?

**Comment by liuzhixing1006** [comment]
Hi @wuchong @JingsongLi , It seems that this pr has met the standard, can you help to merge it? Thank you！

File: docs/dev/table/hive/index.zh.md [hunk 1]
```diff
@@ -294,18 +293,15 @@ You're supposed to add dependencies as stated above at runtime.
 </dependency>
 {% endhighlight %}
 
-## Connecting To Hive
+## 连接到Hive
 
-Connect to an existing Hive installation using the [catalog interface]({{ site.baseurl }}/dev/table/catalogs.html) 
-and [HiveCatalog]({{ site.baseurl }}/dev/table/hive/hive_catalog.html) through the table environment or YAML configuration.
+通过TableEnvironment或者YAML配置，使用 [catalog interface]({{ site.baseurl }}/dev/table/catalogs.html) 和[HiveCatalog]({{ site.baseurl }}/dev/table/hive/hive_catalog.html)连接到现有的Hive集群。
```
Comment (wuchong) [inline docs/dev/table/hive/index.zh.md:298]:
```suggestion
通过TableEnvironment或者YAML配置，使用 [catalog interface]({{ site.baseurl }}/zh/dev/table/catalogs.html) 和[HiveCatalog]({{ site.baseurl }}/zh/dev/table/hive/hive_catalog.html)连接到现有的Hive集群。
```

File: docs/dev/table/hive/index.zh.md [hunk 2]
```diff
@@ -364,8 +360,8 @@ catalogs:
 
 ## DDL
 
-DDL to create Hive tables, views, partitions, functions within Flink will be supported soon.
+即将支持在Flink中创建Hive表，视图，分区和函数的DDL。
 
 ## DML
 
-Flink supports DML writing to Hive tables. Please refer to details in [Reading & Writing Hive Tables]({{ site.baseurl }}/dev/table/hive/read_write_hive.html)
+Flink支持DML写入Hive表，请参考[Reading & Writing Hive Tables]({{ site.baseurl }}/dev/table/hive/read_write_hive.html)
```
Comment (wuchong) [inline docs/dev/table/hive/index.zh.md:367]:
```suggestion
Flink支持DML写入Hive表，请参考[Reading & Writing Hive Tables]({{ site.baseurl }}/zh/dev/table/hive/read_write_hive.html)
```

**Pull request closed without merging**